### PR TITLE
Allow message to be returned in JSON for GET single arrival message e…

### PR DESCRIPTION
**Pull request by sonisn1988**
…ndpoint

### CODE REVIEW

File: test/v2/controllers/V2ArrivalsControllerSpec.scala [hunk 1]
```diff
@@ -906,12 +907,15 @@ class V2ArrivalsControllerSpec
   }
 
   "GET /movements/arrivals/:arrivalId/messages/:messageId " - {
```
Comment (drnaylor-oc) [inline test/v2/controllers/V2ArrivalsControllerSpec.scala:909]: I'd like tests for if the converter fails as well, please.

File: resources/public/api/conf/2.0/application.raml [hunk 2]
```diff
@@ -391,12 +391,10 @@ traits:
...
                         },
-                        "departure":{
-                          "id": "/customs/transits/movements/departures/62f4ebbbf581d4aa"
-                        },
-                        "id": "/customs/transits/movements/departures/62f4ebbbf581d4aa/messages/62f4ebbb765ba8c2",
+                        "id": "62f4ebbb765ba8c2",
+                        "departureId": "62f4ebbbf581d4aa",
                         "received": "2022-08-11T11:44:59.83705",
-                        "messageType": "IE015",
+                        "type": "IE015",
```
Comment (drnaylor-oc) [inline resources/public/api/conf/2.0/application.raml:397]: No, this should be `messageType`, as per https://github.com/hmrc/common-transit-convention-traders/blob/3991baefbe9aa83be97973a9cdc6c58335a6e74e/app/models/response/HateoasDepartureResponseMessage.scala#L40

File: resources/public/api/conf/2.0/application.raml [hunk 3]
```diff
@@ -655,7 +653,30 @@ traits:
...
+                             "href": "/customs/transits/movements/arrivals/62f4ebbbf581d4aa/messages/62f4ebbb765ba8c2"
+                           },
+                           "arrival":{
+                             "href": "/customs/transits/movements/arrivals/62f4ebbbf581d4aa"
+                           }
+                         },
+                         "id": "62f4ebbb765ba8c2",
+                         "arrivalId": "62f4ebbbf581d4aa",
+                         "received": "2022-08-11T11:44:59.83705",
+                         "type": "IE007",
```
Comment (drnaylor-oc) [inline resources/public/api/conf/2.0/application.raml:673]: Same here: https://github.com/hmrc/common-transit-convention-traders/blob/3991baefbe9aa83be97973a9cdc6c58335a6e74e/app/models/response/HateoasArrivalResponseMessage.scala#L40

Comment (sonisn1988) [reply resources/public/api/conf/2.0/application.raml:673]: Liandra has modified it and it is now using HateoasMovementMessageResponse and there it is using  "type"                            -> messageSummary.messageType
.

File: resources/public/api/conf/2.0/application.raml (hunk 2 above)
Comment (sonisn1988) [reply resources/public/api/conf/2.0/application.raml:397]: Liandra has modified it and it is now using HateoasMovementMessageResponse and there it is using  "type"                            -> messageSummary.messageType

Comment (drnaylor-oc) [reply resources/public/api/conf/2.0/application.raml:397]: Yup, you're right, my mistake, it is `type`. Carry on!

File: resources/public/api/conf/2.0/application.raml (hunk 3 above)
Comment (drnaylor-oc) [reply resources/public/api/conf/2.0/application.raml:673]: Yup!

**Review by liandra-giacomine — APPROVED** [review]

**Pull request merged**